sandbox/Popups.qml

```diff
@@ -0,0 +1,38 @@
+import QtQuick 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+
+Column {
+    spacing: 20
+
+    StatusButton {
+        text: "StatusModal"
+        onClicked: modalExample.open()
+    }
+
+    StatusModal {
+            id: modalExample
+        anchors.centerIn: parent
+        visible: true
+        header.image.source: "https://pbs.twimg.com/profile_images/1369221718338895873/T_5fny6o_400x400.jpg"
+        header.title: "Header"
+        header.subTitle: "SubTitle"
+        header.detailsButtonSettings.name: "info"
```

Comment:
Did you verify with Simon that this can be other icons than `info` ?
Is it always going to trigger "details"

Reply:
i just call it details bc i dont know how to call it :) i have some naming problem, if you can give me a great name for this button -  i will be grateful. i will verify.

Reply:
Sure, we can come up with a name, but first we have to know if it's always going to be an "info" icon or if it was just a placeholder in the design